externals/global_params.php

```diff
@@ -79,7 +79,12 @@
      $latest_unstable_source="gnucash-".$latest_unstable.".tar.bz2";
 
      // The URLs, we use:
-     define("DEFAULT_URL", "https://www.gnucash.org/"); // The main webserver
+     if (gethostname()=="code") {
```

Comment:
hostname /may/ return "code.gnucash.org" -- I don't know what it will return specifically.
the `hostname` command-line tool returns the FQDN.

Reply:
[gethostname](https://www.php.net/manual/en/function.gethostname.php) might well return just 'code'. It's apparently implementation and OS dependent. The only way to tell is to apply the patch and then look at the resulting page source in a browser.

Reply:
or we we use something like
`if ((gethostname()=="code") or (gethostname()=="code.gnucash.org"))`